forge/src/main/java/top/theillusivec4/veinmining/veinmining/event/VeinMiningEventsListener.java

```diff
@@ -61,4 +67,20 @@ protected void apply(@Nonnull Void objectIn, @Nonnull ResourceManager resourceMa
       }
     });
   }
+  
+  @EventBusSubscriber
```

Comment:
There doesn't need to be an event bus subscriber or an inner class here since an instance of this class is already being added to the event bus listener, the method in this class can just follow the same syntax as the other methods of the class.